report/3.Theory/SemanticsDescription.tex

```diff
@@ -505,7 +505,7 @@ \subsection{Big-Step Semantic for Expressions}
 
 \subsection{Big-Step Semantic for Statements}
 Statements $Stm$ can change the store, because they can change the value of an existing variable.
-Because of the fact that the category have rewriting rules for declarations, this means that statements can also both change the variable environment $Env_V$ and the function environment $Env_F$.
+Because the category has rewriting rules for declarations, statements can also change the variable environment $Env_V$ and the function environment $Env_F$.
```

Comment:
You removed the wrong word, You removed both, I think I commented to remove also (I might have misspoken)

Reply:
Well lambek commented that I should remove both :D